[2021-07-13 09:52:24,986] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: maintest.testtask 2021-07-13T09:52:23.775444+00:00 [queued]>
[2021-07-13 09:52:25,034] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: maintest.testtask 2021-07-13T09:52:23.775444+00:00 [queued]>
[2021-07-13 09:52:25,035] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-07-13 09:52:25,036] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-07-13 09:52:25,041] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-07-13 09:52:25,064] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): testtask> on 2021-07-13T09:52:23.775444+00:00
[2021-07-13 09:52:25,081] {standard_task_runner.py:52} INFO - Started process 3657 to run task
[2021-07-13 09:52:25,086] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'maintest', 'testtask', '2021-07-13T09:52:23.775444+00:00', '--job-id', '163', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/tablecreation.py', '--cfg-path', '/tmp/tmpy41csol5', '--error-file', '/tmp/tmp2uxnb1_5']
[2021-07-13 09:52:25,087] {standard_task_runner.py:77} INFO - Job 163: Subtask testtask
[2021-07-13 09:52:25,156] {logging_mixin.py:104} INFO - Running <TaskInstance: maintest.testtask 2021-07-13T09:52:23.775444+00:00 [running]> on host a2cbcfd4d6f4
[2021-07-13 09:52:25,241] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=maintest
AIRFLOW_CTX_TASK_ID=testtask
AIRFLOW_CTX_EXECUTION_DATE=2021-07-13T09:52:23.775444+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-07-13T09:52:23.775444+00:00
[2021-07-13 09:52:25,297] {python.py:151} INFO - Done. Returned value was: None
[2021-07-13 09:52:25,316] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=maintest, task_id=testtask, execution_date=20210713T095223, start_date=20210713T095224, end_date=20210713T095225
[2021-07-13 09:52:25,377] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-07-13 09:52:25,421] {local_task_job.py:151} INFO - Task exited with return code 0
